processor/otel/consumer.go

```diff
@@ -408,6 +418,93 @@ func parseSpan(span *tracepb.Span, event *model_span.Event) {
 	event.Labels = labels
 }
 
+func parseErrors(source string, otelSpan *tracepb.Span) []*model_error.Event {
+	var errors []*model_error.Event
+	for _, log := range otelSpan.GetTimeEvents().GetTimeEvent() {
+		var isError, hasMinimalInfo bool
+		var err model_error.Event
+		var logMessage, exMessage, exType string
+		for k, v := range log.GetAnnotation().GetAttributes().GetAttributeMap() {
+			if source == sourceFormatJaeger {
+				switch v := v.Value.(type) {
+				case *tracepb.AttributeValue_StringValue:
+					vStr := v.StringValue.Value
+					switch k {
+					case "error", "error.object":
+						exMessage = vStr
+						hasMinimalInfo = true
+						isError = true
+					case "event":
+						if vStr == "error" { // according to opentracing spec
+							isError = true
+						} else if logMessage == "" {
+							// jaeger seems to send the message in the 'event' field
+							// in case 'event' and 'message' are sent, 'message' is used
+							logMessage = vStr
+							hasMinimalInfo = true
+						}
+					case "message":
+						logMessage = vStr
+						hasMinimalInfo = true
+					case "error.kind":
+						exType = vStr
+						hasMinimalInfo = true
+						isError = true
+					case "level":
+						isError = vStr == "error"
+					}
+				}
+			}
+		}
+		if !isError || !hasMinimalInfo {
+			continue
```

Comment:
I'm a little bit concerned that the "!hasMinimalInfo" case will be hard to debug if we don't know the attributes. Maybe we should log (debug or trace level) when we encounter such a record?